manual/parts/threads.tex

```diff
@@ -458,8 +458,8 @@ \subsection{Debug Exception: Breakpoints and Watchpoints}
 
 \begin{description}
 \item[seL4\_NumHWBreakpoints]: Defines the total number of hardware break
-registers available, of all types available on the hardware platform. On the ARM
-Cortex A7 for example, there are 6 exclusive instruction breakpoint registers,
+registers available, of all types available on the hardware platform. On the Arm
+Cortex~A7 for example, there are 6 exclusive instruction breakpoint registers,
```

Comment:
Arm uses a dash "Cortex-A7" in the naming. In the `amsmath` package there is `\nobreakdash` for this.

Reply:
I'd prefer to fix these separate and not side-track the PR even more..